core/src/main/resources/common-logback.xml

```diff
@@ -1,4 +1,8 @@
 <included>
```

Comment:
So just to be clear, logback.xml does not pick this up by default right? We just need to include it in the sub projectgs we want to get these default logging settings? 

If so, my comment that I left is wrong. This seems to be a sane way to do this for our own internal development purposes while not externally showing this to a user of the library